core/src/main/scala/com/microsoft/azure/synapse/ml/core/utils/SynapsePlatformUtils.scala

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
with these fancy functions we might want to make the python ones depend on these ones

Reply:
Emm Platform.py also contains utils to check if it's databricks platform. But this is specifically for scala and Synapse internal platform. If at some point we want to call the function on Synapse internal platform and in python, we could add them afterwards by accessing jvm through sparkContext.